setup.py

```diff
@@ -12,4 +12,22 @@
     test_suite="tests",
     use_2to3=True,
     requires=['slimit'],
+    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
```

Comment:
I did not add 3.4 here for consistency with troposphere.  But by now it's past its EOL and I'd love to add it as well.

Reply:
Done